assets/src/dashboard/components/cardGridItem/cardPreview.js

```diff
@@ -28,6 +28,7 @@ import {
   ThemeGlobals,
   useFocusOut,
 } from '@web-stories-wp/design-system';
+import { clamp, STORY_ANIMATION_STATE } from '@web-stories-wp/animation';
```

Comment:
Ditto here, let's not use `clamp` from animation package

Reply:
Clamp is also used in edit-story.

Reply:
Yeah `edit-story` has its own `clamp` util but that's OK. This can be duplicated. It's just a one-liner function.

Reply:
Do the two clamp functions do the same thing?

Reply:
Yeah pretty much, just with different function signature. But it's ok for those two to be separate.

Reply:
So should we use the one from animation package or from edit story?

Reply:
Let's copy the one from the animation package to dashboard.

Leaving edit-story as-is

Reply:
We would still need animation as a dependency of dashboard because of `STORY_ANIMATION_STATE`.

<img width="985" alt="Screenshot 2021-07-05 at 13 57 02" src="https://user-images.githubusercontent.com/237508/124474921-06dfd800-dd99-11eb-8559-7e5decda2d0b.png">

Reply:
That's OK. Can be dealt with once we extract story preview into its own package